packages/iris-grid/src/IrisGridUtils.ts

```diff
@@ -75,6 +75,28 @@ type HydratedIrisGridState = Pick<
   metrics: Pick<GridMetrics, 'userColumnWidths' | 'userRowHeights'>;
 };
 
+export interface TableSettings {
+  quickFilters?: readonly [
+    number,
+    {
+      text: string;
+    }
+  ][];
+  advancedFilters?: readonly [
+    number,
+    {
+      options: AdvancedFilterOptions;
+    }
+  ][];
+  inputFilters?: readonly InputFilter[];
+  sorts?: readonly {
+    column: ModelIndex;
+    isAbs: boolean;
+    direction: SortDirection;
```

Comment:
Define this after the types below and then use them instead of redefining the types (`DehydratedQuickFilter`, `DehydratedAdvancedFilter`, `DehydratedSort`)